docs/agile-methodologies.md

```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
+- **User personas:** For Whom? If there are multiple end users, consider making multiple stories.
+- **Ordered Steps:** Write a story for each step in a larger process.
+- **Listen to feedback:** Talk to your users and capture the problem or need in their words.
+- **Time:**  Since stories should be completable in one sprint, stories that might take weeks or months to complete should be broken up into smaller stories or should be considered their own epic.
+
+### Examples:
+>- As a **manager**, I want to **be able to understand my colleagues progress**, so I **can better report our success and failures**. 
+>- As a **customer** , I want **shopping cart feature** so that **I can easily purchase items online**.
+>- As an **manager**, I want to **generate a report** so that **I can understand which departments need more resources**.
+
+## Work meetings
+
+### **Planning Session**
+
+Sprint planning is a timeboxed working session that lasts roughly 1 hour for every week of a sprint. 
+Is an event in the Scrum framework where the team determines the product backlog items they will work on during that sprint and discusses their initial plan for completing those product backlog items.  
+
+This agreement defines the sprint backlog and is based on the team’s velocity or capacity and the length of the sprint.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/2A9rkiIcnVI" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Who is involved?
+
+- **Product Owner:** Identifies the candidate product backlog items and their relative priorities, as well as proposes a sprint goal.
+
+- **Team Members:** Determine how many of the product backlog items they forecast they will be able to complete and determine how they will deliver those product backlog items.
+
+- **Scrum Master:** Typically facilitates sprint planning in order to ensure that the discussion is effective and that there is agreement to the sprint goal and that the appropriate product backlog items are included in the sprint backlog.
+
+### Planning steps
+
+1. Discuss any new information that may impact the plan.
+2. Confirm any currently known issues and concerns and record as appropriate.
+3. Review the definition of **Done** and make any appropriate updates based on technology, skill, or team member changes since the last sprint.
+4. Present proposed product backlog items to consider for the sprint backlog.
+5. Determine the needs, sign up for work, and estimate the work owned.
+6. Product Owner answers clarifying questions and elaborates acceptance criteria.
+7. Confirm any new issues and concerns raised during meeting and record.
+8. Confirm any assumptions or dependencies discovered during planning and record.
+9. ScrumMaster calls for a group consensus on the plan.
+10. Team and Product Owner signal if this is the best plan they can make given what they know right now.
```

Comment:
I think is better "point out" instead of "signal"

Reply:
Oki, thanks!!